src/app/components/osd/osd.component.ts

```diff
@@ -111,13 +113,16 @@ export class OsdComponent implements AfterViewInit, OnDestroy {
   private _page: number;
   @Input() set page(v: number) {
     if (v !== this._page) {
-      this._page = v;
+      this._page = v + 1;
       this.pageChange.next(this._page);
     }
   }
   get page() { return this._page; }
   @Output() pageChange = new EventEmitter<number>();
 
+  // tslint:disable-next-line: no-any
+  @Output() data: EventEmitter<any> = new EventEmitter<any>();
```

Comment:
Type is inferred implicitly. No need to specify it again.

Is this emitter of type any?

What "data" stands for, this variable name is probably too generic?